pkg/uplinkdb/uplinkdb.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uplinkdb
+
+import (
+	"context"
+	"crypto"
+	"time"
+
+	"go.uber.org/zap"
+)
+
+// DB stores bandwidth agreements.
+type DB interface {
+	// CreateAgreement adds a new bandwidth agreement.
+	CreateAgreement(context.Context, string, Agreement) error
```

Comment:
rename to something like "SavePublicKey"

Reply:
Done

Reply:
did you forget to save this?

Reply:
fixed it